app.js

```diff
@@ -195,6 +204,44 @@ function add(req, res, next) {
   }
 }
 
+//preferences
+app.get('/preferences', showPreferences);
+app.post('/preferences', submitPreferences);
+
+function showPreferences(req, res) {
+
+  const email = 'sjonnie@gmail.com'; //Insert the user email of the logged in user here
```

Comment:
Let's not set any data inside of our app. If you wish to define a variable without giving it a value you can just set `const email`. Same goes for line 227.

Reply:
The problem is that we cant yet retrieve the email of the logged in user, so this is only a temporary solution until we can retrieve the logged in user email.

If you have suggestions on how to do this, please feel free to help me.

Reply:
There sure are ways to read the email, but we have put them in place. We can't use a temporary solution to add to the codebase and the `develop` branch because everything has to work there. 

Take a good look at how our current data is made up and what you want to use for this. I'm sure you can do this 👍